src/preferInlineDecoratorRule.ts

```diff
@@ -1,20 +1,39 @@
 import { IOptions, IRuleMetadata, Replacement, RuleFailure, Rules } from 'tslint/lib';
 import { isSameLine } from 'tsutils';
 import { Decorator, Node, PropertyAccessExpression, SourceFile } from 'typescript';
-
 import { NgWalker } from './angular/ngWalker';
 import { getDecoratorName } from './util/utils';
 
+enum Decorators {
+  ContentChild = 'ContentChild',
+  ContentChildren = 'ContentChildren',
+  HostBinding = 'HostBinding',
+  HostListener = 'HostListener',
+  Input = 'Input',
+  Output = 'Output',
+  ViewChild = 'ViewChild',
+  ViewChildren = 'ViewChildren'
+}
+
+type DecoratorKeys = keyof typeof Decorators;
+
+const enumKeys = Object.keys(Decorators);
+
+export const decoratorKeys: ReadonlySet<string> = new Set(enumKeys);
+
 export class Rule extends Rules.AbstractRule {
   static readonly metadata: IRuleMetadata = {
     description: 'Ensures that decorators are on the same line as the property/method it decorates.',
     descriptionDetails: 'See more at https://angular.io/guide/styleguide#style-05-12.',
     hasFix: true,
-    optionExamples: ['true', '[true, "HostListener"]'],
+    optionExamples: [true, [true, Decorators.HostListener]],
```

Comment:
What's happen when we use only `true` ? Does that make sense ?

Reply:
It'll report a warn for every decorator. The list is a blacklist. In this example, you'll not receive any failure about `@HostListener`.